### PR TITLE
Feature/update vertical domain

### LINKED ISSUE #334: [FEATURE] Expand the chart's vertical domain so that top positioning elements are not clipped.
**Is your feature request related to a problem?**
Expand the chart's vertical domain so that top positioning elements are not clipped.

**Do you have any additional context?**
chart top line

### PR DESCRIPTION
**Pull request by jovancacvetkovic**
### Description
Makes the Y domain a bit bigger so that top items don't get cropped.

### Issues Resolved
Closes #334

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/Overview/utils/helpers.ts [hunk 1]
```diff
@@ -121,6 +123,21 @@ export function getVisualizationSpec(description: string, data: any, layers: any
   };
 }
 
+export const getYDomainRange = (data: any[], timeUnit: string): number[] => {
+  data = data.filter((item) => item.finding === 1);
+
+  let dateFormat = 'mm';
+  const timeUnitSize = timeUnit.match(/.*(seconds|minutes|hours|date|month|year)$/);
+  if (timeUnitSize) {
+    if (timeUnitSize && timeUnitSize[1]) {
```
Comment (amsiglan) [inline public/pages/Overview/utils/helpers.ts:132]: checking for `timeUnitSize` twice?

File: public/pages/Overview/utils/helpers.ts [hunk 2]
```diff
@@ -121,6 +123,21 @@ export function getVisualizationSpec(description: string, data: any, layers: any
...
+      dateFormat = `${timeUnitSize[1][0]}${timeUnitSize[1][0]}`;
```
Comment (amsiglan) [inline public/pages/Overview/utils/helpers.ts:133]: `${timeUnitSize[1][0]}${timeUnitSize[1][0]}` -- are both the indices supposed to be same?

File: public/pages/Overview/utils/helpers.ts [hunk 3]
```diff
@@ -121,6 +123,21 @@ export function getVisualizationSpec(description: string, data: any, layers: any
   };
 }
 
+export const getYDomainRange = (data: any[], timeUnit: string): number[] => {
```
Comment (amsiglan) [inline public/pages/Overview/utils/helpers.ts:126]: We should write JSDoc comment above this method to explain why we need this?

File: public/pages/Overview/utils/helpers.ts (hunk 1 above)
Comment (jovancacvetkovic) [reply public/pages/Overview/utils/helpers.ts:132]: My bad, fixed.

File: public/pages/Overview/utils/helpers.ts (hunk 2 above)
Comment (jovancacvetkovic) [reply public/pages/Overview/utils/helpers.ts:133]: Yes, these format goes like `hh` or `mm`.

File: public/pages/Overview/utils/helpers.ts (hunk 3 above)
Comment (jovancacvetkovic) [reply public/pages/Overview/utils/helpers.ts:126]: Added.

**Comment by AWSHurneyt** [comment]
As mentioned in [this post](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/404#issuecomment-1423131201), there is an issue with the windows runner that likely causing the windows cypress tests to timeout.

**Review by amsiglan — APPROVED** [review]

**Review by AWSHurneyt — APPROVED** [review]

**Pull request merged**